prover/src/matrix/row_matrix.rs

```diff
@@ -180,24 +180,45 @@ impl<E: FieldElement> RowMatrix<E> {
     /// * A vector commitment is computed for the resulting vector using the specified vector
     ///   commitment scheme.
     /// * The resulting vector commitment is returned as the commitment to the entire matrix.
-    pub fn commit_to_rows<H, V>(&self) -> V
+    pub fn commit_to_rows<H, V>(&self, num_partitions: usize) -> V
```

Comment:
One other alternative (and I'm not sure if it is better) is to have two separate methods: the original one (which would assume partition size 1) and a new one which could look like so:
```rust
pub fn commit_to_rows_partitioned<H, V>(&self, num_partitions: usize) -> V {
    ...
}
```
Then the caller could decide which one to call.

Reply:
I have kept with only one implementation but I don't feel strongly about it, let me know if you would still prefer adding another method